src/Microsoft.IdentityModel.Logging/ISafeLogSecurityArtifact.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.IdentityModel.Logging
+{
+    /// <summary>
+    /// Interface that defines methods to log security artifact.
+    /// </summary>
+    public interface ISafeLogSecurityArtifact
+    {
+        /// <summary>
+        /// Returns a string that represents the disarmed security artifact.
```

Comment:
>disarmed [](http://example.com/codeflow?start=49&length=8)

what is a disarmed security artifact? and what is an armed security artifact? ... and also we are explaining the method names with words that are even harder to understand. Can we pls. simplify the message and also, can we simplify the names of these 2 methods - it's not logical to me what these 2 things are doing?